test/framework/src/test/java/org/elasticsearch/test/rest/yaml/section/DoSectionTests.java

```diff
@@ -51,7 +52,7 @@ public void testWarningHeaders() {
         }
 
         final String testHeader = HeaderWarning.formatWarning("test");
-        final String anotherHeader = HeaderWarning.formatWarning("another");
+        final String anotherHeader = HeaderWarning.formatWarning("another \"with quotes and \\ backslashes\"");
```

Comment:
no actual changes here...just realized we didn't have any coverage for escaped character matching.